src/common/transformations/src/transformations/utils/utils.cpp

```diff
@@ -340,6 +340,19 @@ bool can_eliminate_eltwise_node(const std::shared_ptr<Node>& eltwise,
     }
     return true;
 }
+
+float cast_eps_to_float(double eps_d) {
+    auto eps_f = static_cast<float>(eps_d);
+    if (eps_d > 0.) {  // zero is fine; negative values have no sense
+        if (std::nextafter(eps_d, 0) < static_cast<double>(std::numeric_limits<float>::min()))
```

Comment:
Add unit test for these corner cases and maybe some standard cast value?

Maybe use `ov::cmp::lt`, it should do cast inside.

```suggestion
        if (ov::cmp::lt(std::nextafter(eps_d, 0.), std::numeric_limits<float>::min()))
```

Reply:
The eps values are already tested by mvn transformation tests and group norm decomposition (additional added).
Keeping existing version for readability.

Reply:
The purpose of the ov::cmp::lt is safe compare between different types.